package.json

```diff
@@ -33,5 +34,9 @@
   "dependencies": {
     "node-static": "^0.7.11",
     "portfinder": "^1.0.32"
+  },
+  "volta": {
+    "node": "20.5.0",
+    "npm": "9.8.0"
   }
-}
```

Comment:
Several files were missing an empty return at the end. Git and Github need this for EOL.